tfhe/src/typed_api/mod.rs

```diff
@@ -0,0 +1,51 @@
+#![allow(unused_doc_comments)]
+#![cfg_attr(fmt, rustfmt::skip)]
+#![cfg_attr(doc, feature(doc_auto_cfg))]
+#![cfg_attr(doc, feature(doc_cfg))]
```

Comment:
if it makes sense for other modules of the crate then I think we can use that elsewhere

Reply:
Ideally this should be in the lib.rs as it was in concrete so it applies to the whole lib